Website/sitecore/shell/client/Applications/WeBlog/Import.js

```diff
@@ -0,0 +1,138 @@
+﻿define(["sitecore"], function (Sitecore) {
+    var WordPressImport = Sitecore.Definitions.App.extend({
+        initialized: function () {
+            this.tabManager.showSelectFile(this);
+            this.DataSource.add({
+                BlogName: "",
+                BlogEmail: "",
+                ParentId: "{0DE95AE4-41AB-4D01-9EB0-67441B7C2450}",
+                DataSourceId: "",
+                ImportPosts: true,
+                ImportCategories: true,
+                ImportComments: true,
+                ImportTags: true
+            });
+
+            this.UploadButton.on("click", this.uploadButtonClick, this);
+            this.btnAcceptLocation.on("click", this.acceptLocationButtonClick, this);
+            this.btnImport.on("click", this.importButtonClick, this);
+            this.Uploader.on("change:totalFiles", this.addedFile, this);
```

Comment:
Don't add click event handlers in code. Specify them in the click property of the component (in the rocks UI designer) instead.